src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -247,7 +247,7 @@ inputSource, new PropertiesExpander(System.getProperties()),
                 throw new CheckstyleException(
                         "First module should be Checker, but was " + configName);
             }
-            handleXmlConfig(testInputConfigBuilder, inputFilePath, xmlConfig.getChildren());
+            handleXmlConfig(testInputConfigBuilder, inputFilePath, xmlConfig);
```

Comment:
removed the `getChildren()` so that we can also add the properties of `Checker` module to the `testInputConfigBuilder` instead of directly adding the children module's properites.